src/integration/transaction/application/types/creation-data.ts

```diff
@@ -0,0 +1,30 @@
+import BigNumber from 'bignumber.js';
+import { Masternode } from 'src/integration/masternode/domain/entities/masternode.entity';
+
+export interface MasternodeBaseData {
+  ownerWallet: string;
+  accountIndex: number;
+}
+
+export interface SendFromLiqData extends MasternodeBaseData {
+  to: string;
+  amount: BigNumber;
```

Comment:
I don't like BigNumbers 🤣

Reply:
Then stay with rookie numbers...
No, imho they make sense. and we anyway need somewhere to go from number to BigNumber. Why not at the start?

Reply:
Ok, then let's keep it like this 😉